.gitignore

```diff
@@ -0,0 +1,16 @@
+### Node modules ###
+/node_modules
+
+### Logs ###
+/logs
+
+### Database ###
+/data
+
+### Specific files ###
+npm-debug.log
+
+### WebStorm ###
```

Comment:
In fact, I'll use a more markdown format using '#' for the first level, '##' for the second level and so on ;)